docs/upgrading/general.md

```diff
@@ -16,7 +16,10 @@ This means updates are super simple in the docker world. To update your instance
 # Update to the latest dockerhub images
 docker-compose pull
 
-# Apply the updates to your instllation
+# (Optional) Take the stack down - This applies if the docker images has not been updated but you expect plugin updates
+docker-compose down
```

Comment:
why not making things simpler as an overall process ?

- down
- pull
- up
```suggestion
docker-compose down
docker-compose pull
docker-compose up -d
```

Reply:
If you want to go that route I would recommend pull first, then down -> up, to save some downtime on potentially slow connections. I was making it optional because it's not always needed I think. But I can make this change to commit.
```suggestion

# Update images
docker-compose pull

# Take the stack down
docker-compose down

# Bring the stack up
docker-compose up -d
```

Reply:
let's go for this - it's moslty to have a simple workflow - people are always free to play and try things :)
simple is better :P